templates/settings/payment.php

```diff
@@ -7,47 +7,124 @@
+                    $mis_match_methods = array_filter( $mis_match_methods, function ( $key ) use( $profile_methods ) {
+                        return in_array( $key,  $profile_methods);
+                    } );
+
+                    $unused_methods = array_filter( $unused_methods, function ( $key ) use ( $mis_match_methods ) {
+                        $found = false;
+
+                        foreach ( $mis_match_methods as $mis_match_method ) {
+                            $found = $found || ( false !== stripos( $key, $mis_match_method ) );
+                        }
+
+                        return ! $found;
+                    } );
+                    ?>
+                    <ul>
+                    <?php foreach ( $unused_methods as $method_key ) {
+                        $method = dokan_withdraw_get_method( $method_key );
+
+                        if ( ! empty( $method ) ) {
+                            $has_methods_dropdown = true;
+                        } else {
+                            continue;
+                        }
+
+                        ?>
+                        <li>
+                            <a href="<?php echo esc_url(  home_url( "dashboard/settings/payment/manage-" . $method_key ) ); ?>">
+                                <div>
+                                    <img src="<?php echo esc_url( DOKAN_PLUGIN_ASSEST . "/images/dashboard-settings/payment/$method_key.svg" ); ?>" alt="<?php echo esc_attr( $method_key ); ?>" />
+                                    <span> <?php
+                                        //translators: %s: payment method title
+                                        echo sprintf( __( 'Direct to %s', 'dokan-lite' ), esc_html__( $method['title'], 'dokan-lite' ) );
+                                        ?>
+                                    </span>
+                                </div>
+                            </a>
+                        </li>
+                    <?php } ?>
+                    </ul>
+
+                    <?php if ( ! $has_methods_dropdown ) { ?>
+                        <div>
+                            <?php esc_html_e( 'All payment methods are used', 'dokan-lite' ); ?>
+                        </div>
+                    <?php } ?>
+                </div>
+            </div>
+        </div>
+    </div>
+    <ul>
+
+    <?php wp_nonce_field( 'dokan_payment_settings_nonce' );
+    $mis_match_map = [
+        'dokan-stripe-connect' => 'stripe',
+        'dokan-moip-connect'   => 'moip'
+    ];
+
+    foreach ( $methods as $method_key ) {
+        if ( ! isset( $profile_info['payment'][$method_key] ) && ! isset( $profile_info['payment'][$mis_match_map[$method_key]] ) ) {
+            continue;
+        }
 
-    <?php foreach ( $methods as $method_key ) {
         $method = dokan_withdraw_get_method( $method_key );
 
         if ( ! empty( $method ) ) {
             $has_methods = true;
+        } else {
```

Comment:
Update this as mentioned above

Reply:
understood the process, but this will need to store the $method_key somewhere else because $method_key is used to get the icon assets.